msg/tools/uorb_rtps_message_ids.py

```diff
@@ -9,12 +9,11 @@
     'actuator_outputs': 4,
     'adc_report': 5,
     'airspeed': 6,
-    'att_pos_mocap': 7,
-    'battery_status': 8,
-    'camera_capture': 9,
-    'camera_trigger': 10,
-    'collision_report': 11,
-    'commander_state': 12,
+    'battery_status': 7,
```

Comment:
Please do not change the message id's of existing messages, it will break compatibility.

Reply:
OK, I will just add the new ones with a new ID each.